### PR TITLE
C1 khloud c5 s12 report words in game

### PR DESCRIPTION
**Pull request by Khloud-Khalid**
[Issue #360 ]
@adamggg , @KareemAliRahman, @nourhanZ  please review.

### CODE REVIEW

File: arability/app/views/games/_vote.html.erb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+		record_vote(synonym_id,"formal");
+	}
+	function choose_slang_and_vote(){
+		var synonym_id = $("#choosen_synonym_id").val();
+		record_vote(synonym_id,"slang");
+	}
+
+	<%= render "search/submit-report.js" %>
+</script>
+<div class="alert alert-success" id="report_success" style="display:none; font-size: 16px;">
```
Comment (adamggg) [inline arability/app/views/games/_vote.html.erb:129]: Fix indentations =D

File: arability/app/views/games/_vote.html.erb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+		  <p><%= t(:check_connection) %></p>
+	  </div>
+	  <script type="text/javascript">
+			<% if I18n.locale == :ar %>
+				$("#launch_report").css("font-size","18px");
+			<% else %>
+				$("#launch_report").css("font-size","13px");
+			<% end %>
+		</script>
+<div id="report_modal" class="modal hide fade" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true" style="font-size: 16px;">
```
Comment (adamggg) [inline arability/app/views/games/_vote.html.erb:151]: Fix indentations =D

File: arability/app/views/games/_vote.html.erb (hunk 1 above)
Comment (Khloud-Khalid) [inline arability/app/views/games/_vote.html.erb:129]: what's wrong with them? :D

Comment (adamggg) [inline arability/app/views/games/_vote.html.erb:129]: Nothing nevermind =D :octocat:

**Pull request merged**